### ISSUE TITLE
Discord Rich Presence module doesn't allow game to start

### ISSUE THREAD

**Issue by phibluu** [MeteorDevelopment/meteor-client#1579]
### Describe the bug

Using the meteor client download from the website the game crashes almost instantly, log: https://paste.ee/p/GT5mH
After reading through the log and a little bit of searching I found out the problem was the discord-rpc module. So I went through the code and deleted all instances of Discord-RPC that showed up in the log in the sources. And only after doing this and building from source did meteor client launch.

Extra information:
OS: MacOS Big Sur 11.6 on M1
I used MultiMC to run the game natively on arm

I would appreciate any help because though you don't need discord rich presence it's very nice to have,
Thanks

### Steps to reproduce

```bash
By simply launching the game without a modified build makes it crash instantly.
```

### Meteor Version

0.4.6-1292

### Minecraft Version

1.17.1

### Operating System

macOS

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by colourfullyy** [comment]
Well it seems the problem is that I'm on arm, because it launches and Discord Rich Presence works on an x64 jdk.
